src/main/app/form/validation.ts

```diff
@@ -73,16 +73,17 @@ export const isFutureDate: DateValidator = date => {
   }
 };
 
-export const isLessThanAYear: DateValidator = date => {
+export const isLessThanAYearAgoInc: DateValidator = date => {
   if (!date) {
     return;
   }
 
   const enteredDate = new Date(+date.year, +date.month - 1, +date.day);
-  const oneYearAgo = new Date();
+  const today = new Date();
+  const oneYearAgo = new Date(today.getFullYear(), today.getMonth(), today.getDate());
   oneYearAgo.setFullYear(oneYearAgo.getFullYear() - 1);
-  if (!(enteredDate < oneYearAgo)) {
-    return 'lessThanAYear';
+  if (enteredDate >= oneYearAgo) {
+    return 'lessThanAYearAgoInc';
```

Comment:
I know this is how it was before, but to me it looks like the validators should only return a string when there is an error, not when the validation is successful. because otherwise they should return a boolean, but the string return is to tell us what type of error it is

Reply:
Yes - I tried it the other way around, and it caused a lot of problems - see my comments on the JIRA